pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -269,7 +277,8 @@ func makeMounts(opts *kubecontainer.RunContainerOptions, container *api.Containe
 		// here we just add a random id to make the path unique for different instances
 		// of the same container.
 		cid := makeUID()
-		containerLogPath := path.Join(opts.PodContainerDir, cid)
+		containerLogPath := filepath.Join(opts.PodContainerDir, cid)
+		// TODO: We should try to use os interface here.
 		fs, err := os.Create(containerLogPath)
```

Comment:
A small problem is that `makeMounts` is not a member function of `m`. We can easily change that, just don't know whether we should make this change just for the unit test.